relayer/relays/beacon/parachain-writer.go

```diff
@@ -128,9 +128,9 @@ func (wr *ParachainWriter) getLastSyncedSyncCommitteePeriod() (uint64, error) {
 	}
 
 	var period types.U64
-	ok, err := wr.conn.API().RPC.State.GetStorageLatest(key, &period)
-	if err != nil || !ok {
-		return 0, fmt.Errorf("get storage for latest synced sync committee period: %w", err)
+	_, err = wr.conn.API().RPC.State.GetStorageLatest(key, &period)
```

Comment:
Do we need this check if the result is not used?

Reply:
Yes, we do. If there is an error the err variable is checked. The `ok` variable is false if the returned int is `0`, which is valid sync period value. So that's why I removed checking the `ok` var.